solvebio/resource/vault.py

```diff
@@ -0,0 +1,70 @@
+"""Solvebio Vault API resource"""
+# from ..client import client
+from ..help import open_help
+
+from .apiresource import CreateableAPIResource
+from .apiresource import ListableAPIResource
+from .apiresource import SearchableAPIResource
+from .apiresource import UpdateableAPIResource
+from .apiresource import DeletableAPIResource
+
+
+class Vault(CreateableAPIResource,
```

Comment:
Can all these actions be done?

* Get object in Vault by ID
* Get object by path
* List objects from a search